### DISCUSSION TITLE
Update bug that breaks the system

### DISCUSSION THREAD

**Discussion by jay77777777** [rern/rAudio#458] [Fixed Issues]
New install. Updated to 8-7-31 and added a network share. Accidently disconnected the power during the update. Saw that update continued after I restored power. This morning it was still updating. Tried removing the network share and restarting. Still stuck in updating mode and never finishes. System broken.  I even did a system restore and it still continues trying to update in vain. Only fix will be to reimage card.

**Comment by rern** [comment]
I wish rAudio could be that robust which it could survive power loss in the middle of a crucial process. 
I've tried disconnecting power supply while idle a few times and it's ok. But I don't think it's gonna make it while updating.

**Reply by jay77777777** [reply]
You're right.  What I encountered is an unusual scenario that most users will likely never run into.
